tests/store/tracking/test_gateway_sql_store.py

```diff
@@ -198,6 +198,38 @@ def test_list_secret_infos(store: SqlAlchemyStore):
     assert all(s.provider == "openai" for s in openai_secrets)
 
 
+def test_secret_id_and_name_are_immutable_at_database_level(store: SqlAlchemyStore):
```

Comment:
Could we run this test file in database job https://github.com/mlflow/mlflow/blob/312edd5feb63a274e762cdde3c85f9e2bb074643/.github/workflows/master.yml#L132-L135?

Reply:
Great idea!! Updated CI config :D